### PR TITLE
WIP (SIMP-6431) Updated static assets

### PR DESCRIPTION
**Pull request by sharkbruhaha**
close #SIMP-6431

### CODE REVIEW

**Comment by sharkbruhaha** [comment]
https://gitlab.com/simp/pupmod-simp-acpid/pipelines/58064724

File: .travis.yml [hunk 1]
```diff
@@ -106,15 +81,15 @@ jobs:
       deploy:
         - provider: releases
           api_key:
-              secure: "LkPgCyPkenRqkuOWdZOkVKu8jIfWcVSknF+HJUOr+B8r6tV0FdWQPgpb6h6T4ll/kjA7y+/y0aMkvPRpsGecxVEdC7y0MOgk+WFuG24BwF+xTtEGlYtqK/2LwSqoJ8Cw98L6qtWsncbWn4Zn2kyiaQnqOC36BDQTsfLY/Z+BoABmi334Hz06SXEHFjxECFw/R38ngUoHaoimdu1JeXybyqIFneoq0WOiv5o+9yQIAx4tBAPMS17SlQxAZbf5b/zz6KGzvnykvFeLu+hMyf5+fsAaOs13ImhUhbIEtZpu9+e5aHFt5RvtDjIrVXqAF3/+Tk35JDwR9tYKS+KPXjaGeXgwOmhMeJf52sJm6GRwajldh4tpJQNCoCCHUtEh+BR39KfYyu+2FUhETffy3isYzc1+BZzddGpwKOlEiD/2GKSoxQ2ILBEhW77Y1viDj8IMsE4PX40DPtE5F9+8JCT0KDHvQoLLgRktuO1d7bgJX1vbaEPH6y8t5V06dLHyjNa+bBUS4IcvNk+8km2bjtxsIp4o8dKG6QH5xvCUIL5SaUPZHrrykBTrTlWBvNrmx7iQt6oSA3arLEIw5y/GG5HMPieUSgNTWemCEL0KbvtJlRXFHwK6TAQmHsfkzcnUu2Raa4lIdITxTtJXVfW/t5HIqRfo/75OeGJvXX3vfDbJQ8k="
+            secure: "CHANGE_SECURE"
```
Comment (trevor-vaughan) [inline .travis.yml]: These must be preserved

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -5,14 +5,11 @@
 # https://puppet.com/docs/pe/2018.1/overview/getting_support_for_pe.html
```
Comment (trevor-vaughan) [inline .gitlab-ci.yml]: Can't randomly copy over this file. The acceptance test portion must stay intact and may vary per module.

The unit testing and linting portions can be updated consistently.

File: .travis.yml (hunk 1 above)
Comment (sharkbruhaha) [reply .travis.yml]: Was not sure if all needed to be updated or not, as I was told a number were out of date or not working any longer.

File: .gitlab-ci.yml (hunk 2 above)
Comment (sharkbruhaha) [reply .gitlab-ci.yml]: roger

File: .travis.yml (hunk 1 above)
Comment (trevor-vaughan) [reply .travis.yml]: @sharkbruhaha Well....until we know they don't work we shouldn't erase them. I couldn't find a tracking ticket in JIRA to note the ones that are failing.

Comment (sharkbruhaha) [reply .travis.yml]: is there a way to easily test the keys without pushing them to the forge/releasing them? If so I can start work on that and populate a ticket for that.

Comment (trevor-vaughan) [reply .travis.yml]: Nope, there's not! It's a bunch of fun.

Comment (sharkbruhaha) [reply .travis.yml]: haha, ok then. Will just place back keys for each PR once submitted and that future ticket can be worked out at a later date.

**Comment by trevor-vaughan** [comment]
Suggest moving to [include](https://docs.gitlab.com/ee/ci/yaml/#includefile) and [extends](https://docs.gitlab.com/ee/ci/yaml/#extends) for our `gitlab.ci-yml` and splitting the various parts into individual local files under a `.gitlab-ci` directory.

**Comment by sharkbruhaha** [comment]
Shifting this to a WIP while I try to integrate the include/extends functionality

**Comment by sharkbruhaha** [comment]
https://gitlab.com/simp/pupmod-simp-acpid/pipelines/59658690

**Comment by trevor-vaughan** [comment]
Decided to take a different tack on the assets moving forward.

**Pull request closed without merging**